manifests/plugin/puppetdb.pp

```diff
@@ -13,10 +13,25 @@
 # $dashboard_address:: Address of puppetdb dashboard. Defaults to 'http://localhost:8080/dashboard'
 #                      type:Stdlib::HTTPUrl
 #
+# $ssl_ca_file:: CA certificate file which will be used to connect PuppetDB API.
+#                Defaults to client_ssl_ca
+#                type:String
+#
+# $ssl_certificate:: Certificate file which will be used to connect PuppetDB API.
+#                Defaults to client_ssl_cert
+#                type:String
+#
+#
+# $ssl_private_key:: Private key file which will be used to connect PuppetDB API.
+#                Defaults to client_ssl_key
+#                type:String
 class foreman::plugin::puppetdb (
   $package           = $foreman::plugin::puppetdb::params::package,
   $address           = $foreman::plugin::puppetdb::params::address,
   $dashboard_address = $foreman::plugin::puppetdb::params::dashboard_address,
+  $ssl_ca_file       = $foreman::plugin::puppetdb::params::client_ssl_ca,
```

Comment:
The names here don't match, causing test failures (it's called $ssl_ca_file in params, not client_ssl_ca.)